packages/graph/hash_graph/bench/benches/read_scaling/knowledge/linkless/mod.rs

```diff
@@ -0,0 +1,7 @@
+/// An "edgeless" graph is a graph without any edges.
+///
+/// These benchmarks are for a datastore where the knowledge-graph component is an edgeless graph,
+/// meaning there are no link entities, hence the name "linkless".
```

Comment:
Just to note, I don't think we can do an "edgeless" graph easily, because every entity should have an entity type.. and in turn those probably should have property types and data types to actually have any meaningful size of data, so.. I think `linkless` is testing the case we care about